replay/config.ini

```diff
@@ -3,12 +3,13 @@
 ip=127.0.0.1
 port=8090
 insecure=false
-token=../../kuksa_certificates/jwt/all-read-write.json.token
+token=../../kuksa.val/kuksa_certificates/jwt/all-read-write.json.token
```

Comment:
I changed the example/default config so that it works if you have kuksa.val.feeders and kuksa.val cloned in parallel.